SmartHomeManager/SmartHomeManager.View/src/pages/DeviceConfig.js

```diff
@@ -0,0 +1,88 @@
+import React, { useState } from "react";
+import { Select } from '@chakra-ui/react';
+import {
+    Slider,
+    SliderTrack,
+    SliderFilledTrack,
+    SliderThumb,
+    SliderMark,
+    Box
+} from '@chakra-ui/react'
+
+export default function DeviceConfig() {
+
+    const [DropDownValue, setDropDownValue] = useState(null);
+    const [sliderValue, setSliderValue] = useState(50)
+
+
+    const labelStyles = {
+        mt: '2',
+        ml: '-2.5',
+        fontSize: 'sm',
+    }
+
+    const dropDownHandler = (e) => {
+        let value = e.target.value;
+        setDropDownValue(value);
+        if (value === "option1") {
```

Comment:
@rawsashimi1604 be careful, would just be better to set as value === 50/30 instead